ui/temboardui/templates/flask/settings/users.html

```diff
@@ -44,24 +43,22 @@
             <td>{{role_item.role_name}}</td>
             <td>{{role_item.role_email or ''}}</td>
             <td>{{role_item.role_phone or ''}}</td>
-            <td class="text-center" data-col="is-active">{% if role_item.is_active is True %}<span class="badge text-bg-success">Yes</span>{% else %}<span class="badge text-bg-danger">No</span>{% end %}</td>
-            <td class="text-center" data-col="is-admin">{% if role_item.is_admin is True %}<span class="badge text-bg-success">Yes</span>{% else %}<span class="badge text-bg-danger">No</span>{% end %}</td>
-            <td>{% for group in role_item.groups %} <span class="badge border text-body-secondary">{{group.group_name}}</span>{% end %}</td>
+            <td class="text-center" data-col="is-active">{% if role_item.is_active %}<span class="badge text-bg-success">Yes</span>{% else %}<span class="badge text-bg-danger">No</span>{% endif %}</td>
+            <td class="text-center" data-col="is-admin">{% if role_item.is_admin %}<span class="badge text-bg-success">Yes</span>{% else %}<span class="badge text-bg-danger">No</span>{% endif %}</td>
+            <td>{% for group in role_item.groups %} <span class="badge border text-body-secondary">{{group.group_name}}</span>{% endfor %}</td>
             <td class="text-center">
               <button
                 type="button"
-                class="btn btn-outline-secondary btn-sm"
+                class="btn btn-outline-secondary btn-sm m-1"
```

Comment:
Not completely related to migration to flask.

Reply:
Pareil, harmonisation.